### PR TITLE
Wait screen

### LINKED ISSUE #428: Display spinner screen after password input and before `lndReady` is true

### PR DESCRIPTION
**Pull request by tanx**
Just a super simple spinner while lnd is initializing for the alpha release. I'll ask Lee if he can design something nicer.

Closes #428

### CODE REVIEW

File: src/action/nav.js [hunk 1]
```diff
@@ -31,6 +31,10 @@ class NavAction {
     this._store.route = 'NewAddress';
   }
 
+  goWait() {
```
Comment (valentinewallace) [inline src/action/nav.js]: I guess it doesn't matter if it's really temporary, but I'd go with a more descriptive name like `waitForNetworkLoad`. Unless we plan on having 1 `Wait` screen for multiple types of waiting?

File: src/view/wait.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+const styles = StyleSheet.create({
+  content: {
+    justifyContent: 'center',
+  },
+  spinner: {
+    transform: [{ scale: 1.5 }],
+  },
+});
+
+const WaitView = () => (
```
Comment (valentinewallace) [inline src/view/wait.js]: Lol, looks good :ok_hand:

File: src/view/wait.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  <Background color={color.blackDark}>
+    <MainContent style={styles.content}>
+      <ActivityIndicator
+        size="large"
+        color={color.lightPurple}
+        style={styles.spinner}
+      />
```
Comment (valentinewallace) [inline src/view/wait.js]: I might add a `copy` section with a small description of what the user is waiting on.

File: src/action/nav.js (hunk 1 above)
Comment (tanx) [reply src/action/nav.js]: Yes. I also added the wait screen for the lightning payment. That takes a few seconds on testnet

File: src/view/wait.js (hunk 2 above)
Comment (tanx) [reply src/view/wait.js]: Right. Engineer designed UI ftw :)

File: src/view/wait.js (hunk 3 above)
Comment (tanx) [reply src/view/wait.js]: Would make sense if we used it only on one screen. We're going to replace this anyway with a fancier design anyway. So I think it's ok for now.

**Comment by tanx** [comment]
Merging for now. We can always iterate on the design once we have an update from Lee.

**Pull request merged**